RecoEgamma/PhotonIdentification/python/Identification/cutBasedPhotonID_Spring16_V2p2_cff.py

```diff
@@ -0,0 +1,156 @@
+    0.0148    ,  # absPFNeuHadIsoWithEACut_C2
+    0.000017  ,  # absPFNeuHadIsowithEACut_C3
+    2.362     ,  # absPFPhoIsoWithEACut_C1
+    0.0047       # absPFPhoIsoWithEACut_C2
+    )
+
+WP_Tight_EE = WorkingPoint_V2(
+    idName    ,  #idName
+    0.0213    ,  # hOverECut
+    0.03000   ,  # full5x5_SigmaIEtaIEtaCut
+# Isolation cuts are generally absIso < C1 + pt*C2, except for NeuHad is < C1 + pt*C2 + pt*pt*C3
+    0.034     ,  # absPFChaHadIsoWithEACut_C1
+    0.00      ,  # absPFChaHadIsoWithEACut_C2
+    0.586     ,  # absPFNeuHadIsoWithEACut_C1
+    0.0163    ,  # absPFNeuHadIsoWithEACut_C2
+    0.000014  ,  # absPFNeuHadIsowithEACut_C3    
+    2.617     ,  # absPFPhoIsoWithEACut_C1
+    0.0034       # absPFPhoIsoWithEACut_C2
+    )
+
+
+# Second, define where to find the precomputed isolations and what effective
+# areas to use for pile-up correction
+isoInputs = IsolationCutInputs(
+    # chHadIsolationMapName  
+    'photonIDValueMapProducer:phoChargedIsolation' ,
+    # chHadIsolationEffAreas 
+    "RecoEgamma/PhotonIdentification/data/Spring16/effAreaPhotons_cone03_pfChargedHadrons_90percentBased.txt",
+    # neuHadIsolationMapName
+    'photonIDValueMapProducer:phoNeutralHadronIsolation' ,
+    # neuHadIsolationEffAreas
+    "RecoEgamma/PhotonIdentification/data/Spring16/effAreaPhotons_cone03_pfNeutralHadrons_90percentBased.txt" ,
+    # phoIsolationMapName  
+    "photonIDValueMapProducer:phoPhotonIsolation" ,
+    # phoIsolationEffAreas
+    "RecoEgamma/PhotonIdentification/data/Spring16/effAreaPhotons_cone03_pfPhotons_90percentBased.txt"
+)
+
+#
+# Finally, set up VID configuration for all cuts
+#
+cutBasedPhotonID_Spring16_V2p2_loose  = configureVIDCutBasedPhoID_V5 ( WP_Loose_EB, WP_Loose_EE, isoInputs)
+cutBasedPhotonID_Spring16_V2p2_medium = configureVIDCutBasedPhoID_V5 ( WP_Medium_EB, WP_Medium_EE, isoInputs)
+cutBasedPhotonID_Spring16_V2p2_tight  = configureVIDCutBasedPhoID_V5 ( WP_Tight_EB, WP_Tight_EE, isoInputs)
+
+## The MD5 sum numbers below reflect the exact set of cut variables
+# and values above. If anything changes, one has to 
+# 1) comment out the lines below about the registry, 
+# 2) run "calculateMD5 <this file name> <one of the VID config names just above>
+# 3) update the MD5 sum strings below and uncomment the lines again.
+#
+
+central_id_registry.register(cutBasedPhotonID_Spring16_V2p2_loose.idName,
+                             'd6ce6a4f3476294bf0a3261e00170daf')
+central_id_registry.register(cutBasedPhotonID_Spring16_V2p2_medium.idName,
+                             'c739cfd0b6287b8586da187c06d4053f')
+central_id_registry.register(cutBasedPhotonID_Spring16_V2p2_tight.idName,
+                             'bdb623bdb1a15c13545020a919dd9530')
+
+cutBasedPhotonID_Spring16_V2p2_loose.isPOGApproved = cms.untracked.bool(True)
+cutBasedPhotonID_Spring16_V2p2_medium.isPOGApproved = cms.untracked.bool(True)
+cutBasedPhotonID_Spring16_V2p2_tight.isPOGApproved = cms.untracked.bool(True)
```

Comment:
the following warnings are showing up regularly now in 93X (from around 930pre1)

```
%MSG-w IdInformation:  VersionedPhotonIdProducer:egmPhotonIDs@ctor  18-Jul-2017 18:30:59 CEST pre-events
Instantiated ID: cutBasedPhotonID-Spring16-V2p2-loose
with MD5 hash: d6ce6a4f3476294bf0a3261e00170daf
This ID is not POG approved and likely under development!!!
Please make sure to report your progress with this ID at the next relevant POG meeting.

%MSG
%MSG-w IdInformation:  VersionedPhotonIdProducer:egmPhotonIDs@ctor  18-Jul-2017 18:30:59 CEST pre-events
Instantiated ID: cutBasedPhotonID-Spring16-V2p2-medium
with MD5 hash: c739cfd0b6287b8586da187c06d4053f
This ID is not POG approved and likely under development!!!
Please make sure to report your progress with this ID at the next relevant POG meeting.

%MSG
%MSG-w IdInformation:  VersionedPhotonIdProducer:egmPhotonIDs@ctor  18-Jul-2017 18:30:59 CEST pre-events
Instantiated ID: cutBasedPhotonID-Spring16-V2p2-tight
with MD5 hash: bdb623bdb1a15c13545020a919dd9530
This ID is not POG approved and likely under development!!!
Please make sure to report your progress with this ID at the next relevant POG meeting.
```
something is wrong with the hashes